examples/tutorials/audio_data_augmentation_tutorial.py

```diff
@@ -22,149 +27,157 @@
 # --------------------------------------------------------
 #
 
-#@title Prepare data and utility functions. {display-mode: "form"}
-#@markdown
-#@markdown You do not need to look into this cell.
-#@markdown Just execute once and you are good to go.
-#@markdown
-#@markdown In this tutorial, we will use a speech data from [VOiCES dataset](https://iqtlabs.github.io/voices/), which is licensed under Creative Commos BY 4.0.
+# @title Prepare data and utility functions. {display-mode: "form"}
+# @markdown
+# @markdown You do not need to look into this cell.
+# @markdown Just execute once and you are good to go.
+# @markdown
+# @markdown In this tutorial, we will use a speech data from [VOiCES dataset](https://iqtlabs.github.io/voices/),
+# @markdown which is licensed under Creative Commos BY 4.0.
 
-#-------------------------------------------------------------------------------
+# -------------------------------------------------------------------------------
 # Preparation of data and helper functions.
-#-------------------------------------------------------------------------------
-
-import math
-import os
-import requests
-
-import matplotlib.pyplot as plt
-from IPython.display import Audio, display
+# -------------------------------------------------------------------------------
 
 
 _SAMPLE_DIR = "_assets"
 
 SAMPLE_WAV_URL = "https://pytorch-tutorial-assets.s3.amazonaws.com/steam-train-whistle-daniel_simon.wav"
 SAMPLE_WAV_PATH = os.path.join(_SAMPLE_DIR, "steam.wav")
 
-SAMPLE_RIR_URL = "https://pytorch-tutorial-assets.s3.amazonaws.com/VOiCES_devkit/distant-16k/room-response/rm1/impulse/Lab41-SRI-VOiCES-rm1-impulse-mc01-stu-clo.wav"
+SAMPLE_RIR_URL = "https://pytorch-tutorial-assets.s3.amazonaws.com/VOiCES_devkit/"\
+    "distant-16k/room-response/rm1/impulse/Lab41-SRI-VOiCES-rm1-impulse-mc01-stu-clo.wav"
 SAMPLE_RIR_PATH = os.path.join(_SAMPLE_DIR, "rir.wav")
 
-SAMPLE_WAV_SPEECH_URL = "https://pytorch-tutorial-assets.s3.amazonaws.com/VOiCES_devkit/source-16k/train/sp0307/Lab41-SRI-VOiCES-src-sp0307-ch127535-sg0042.wav"
+SAMPLE_WAV_SPEECH_URL = "https://pytorch-tutorial-assets.s3.amazonaws.com/VOiCES_devkit/"\
+    "source-16k/train/sp0307/Lab41-SRI-VOiCES-src-sp0307-ch127535-sg0042.wav"
 SAMPLE_WAV_SPEECH_PATH = os.path.join(_SAMPLE_DIR, "speech.wav")
 
-SAMPLE_NOISE_URL = "https://pytorch-tutorial-assets.s3.amazonaws.com/VOiCES_devkit/distant-16k/distractors/rm1/babb/Lab41-SRI-VOiCES-rm1-babb-mc01-stu-clo.wav"
+SAMPLE_NOISE_URL = "https://pytorch-tutorial-assets.s3.amazonaws.com/VOiCES_devkit/"\
+    "distant-16k/distractors/rm1/babb/Lab41-SRI-VOiCES-rm1-babb-mc01-stu-clo.wav"
```

Comment:
Did you use tool or do it manually? Normally, URL-like strings are exempted from formatter line length rules.

Reply:
I did it manually, to solve D32280635

Reply:
Okay, please use `black` or apply the internal linter.
Considering the number of lines changed, manual change is error-prune and hard to verify.

Long URLs should be exempted from line lengths. Please use  `# noqa: E501` at the end of each line that's too long. See for example 

https://github.com/pytorch/audio/blob/e6dd7fd3ef3e07df92b0f46180b3d5888e59c014/torchaudio/pipelines/_wav2vec2/impl.py#L41-L45